src/docs/development/tools/vs-code.md

```diff
@@ -118,102 +142,95 @@ of your changes almost instantly with the _Stateful Hot Reload_ feature. See
 
 ### Debugging visual layout issues
 
-During a debug session, several additional debugging commands are added to the
+During a debug session,
+several additional debugging commands are added to the
 [Command Palette][], including:
 
-* **Toggle Baseline Painting**: Causes each RenderBox to paint a line at each of its
-  baselines.
-* **Toggle Repaint Rainbow**: Show rotating colors on layers when repainting.
-* **Toggle Slow Animations**: Slow down animations to enable visual inspection.
-* **Toggle Slow-Mode Banner**: Hide the slow mode banner even when running a
+* **Toggle Baseline Painting**: Causes each RenderBox to paint a line
```

Comment:
Might mention these can also be toggled via buttons in the inspector.
Bonus points: include the icons from the inspector next to each icon in this list to use a consistent vocabulary.